docker-compose-ode.yml

```
@@ -57,18 +57,20 @@ services:
       - "5555:5555/udp"
       - "6666:6666/udp"
     environment:
-      DOCKER_HOST_IP: ${DOCKER_HOST_IP:?error}
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      ODE_KAFKA_BROKERS: ${DOCKER_HOST_IP}:9092
       KAFKA_TYPE: ${KAFKA_TYPE}
       CONFLUENT_KEY: ${CONFLUENT_KEY}
       CONFLUENT_SECRET: ${CONFLUENT_SECRET}
-      ODE_TIM_INGEST_MONITORING_ENABLED: true
-      ODE_TIM_INGEST_MONITORING_INTERVAL: 60
-      DATA_SIGNING_ENABLED_RSU: false
-      DATA_SIGNING_ENABLED_SDW: false
-      ODE_KAFKA_BROKERS: ${DOCKER_HOST_IP}:9092
-      KAFKA_LINGER_MS: 1
-      KAFKA_BATCH_SIZE: 16384
-      KAFKA_RETRIES: 0
+      KAFKA_LINGER_MS: ${KAFKA_LINGER_MS:-1}
+      KAFKA_ACKS: ${KAFKA_ACKS:-all}
+      KAFKA_RETRIES: ${KAFKA_RETRIES:-0}
+      KAFKA_BATCH_SIZE: ${KAFKA_BATCH_SIZE:-16384}
+      KAFKA_BUFFER_MEMORY: ${KAFKA_BUFFER_MEMORY:-33554432}
+      KAFKA_COMPRESSION_TYPE: ${KAFKA_COMPRESSION_TYPE:-zstd}
+      KAFKA_KEY_SERIALIZER: ${KAFKA_KEY_SERIALIZER:-org.apache.kafka.common.serialization.StringSerializer}
+      KAFKA_VALUE_SERIALIZER: ${KAFKA_VALUE_SERIALIZER:-org.apache.kafka.common.serialization.StringSerializer}
+      KAFKA_PARTITIONER_CLASS: ${KAFKA_PARTITIONER_CLASS:-org.apache.kafka.clients.producer.internals.DefaultPartitioner}

```

Add new ODE variables
```suggestion
      KAFKA_PARTITIONER_CLASS: ${KAFKA_PARTITIONER_CLASS:-org.apache.kafka.clients.producer.internals.DefaultPartitioner}
      ODE_TIM_INGEST_MONITORING_ENABLED: ${ODE_TIM_INGEST_MONITORING_ENABLED:-false}
      ODE_TIM_INGEST_MONITORING_INTERVAL: ${ODE_TIM_INGEST_MONITORING_INTERVAL:-20}
      ODE_STOMP_EXPORTER_ENABLED: ${ODE_STOMP_EXPORTER_ENABLED:-false}
```

May also require updates to the `sample.env`